server/channels/api4/post.go

```diff
@@ -1267,3 +1272,19 @@ func getPostInfo(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	w.Write(js)
 }
+
+func hasPermittedRole(c *Context, user *model.User, channelMember *model.ChannelMember) bool {
```

Comment:
Can we be a bit more specific on the naming here, since this is specific to Wrangler, something like:
```suggestion
func hasPermittedWranglerRole(c *Context, user *model.User, channelMember *model.ChannelMember) bool {
```

Reply:
Good point. Done.